File: C_C++/Bieu_dien_nhi_phan.cpp
#include <iostream>
using namespace std;
void Bieu_dien_nhi_phan(int a[], unsigned long long &n)
{
    cin >> n;
    unsigned long long temp = n;
    int i=0;
    int count=0;
    while (temp!=0)
    {
        if (temp % 2 == 0)
            a[i] = 0;
        else
            a[i] = 1;
        i++;
        count++;
        temp /=2;
    }
    for (i=count-1;i>=0;i--)
        cout << a[i];
}
int main(int argc, char *argv[])
{
	unsigned long long n;
	int a[100];
	Bieu_dien_nhi_phan(a,n);
}

File: Python/Bieu_dien_nhi_phan.py
def Bieu_dien_nhi_phan(n): 
    _binary = [0] * 100;
    temp = int(n);
    i = int(0)
    count = int(0);
    while(temp!=0):
        if(temp%2==0):
            _binary[i]=0
        else:
            _binary[i]=1;
        temp /= 2
        i+=1;
    print(_binary)

n = 29;
Bieu_dien_nhi_phan(29)